packages/data/tests/helpers/test-data-setup.js

```diff
@@ -116,13 +116,13 @@ export default function(factory, opts) {
 
   let vanGogh = factory.addResource('puppies', 'vanGogh')
     .withAttributes({
-      name: 'Van Gogh',
+      nickname: 'Van Gogh',
```

Comment:
Presumably this is avoiding a collision, but the collision is only a temporary thing until we can refactor all tests to go through the new schema feature?

Reply:
confirm